package/CHANGELOG

```diff
@@ -19,6 +19,8 @@ mm/dd/17 utkbansal
 
 Enhancements
   * Universe now works with StringIO objects for topologies and trajectories.
+  * Residues with the same residue ids are not merged by default now 
```

Comment:
Also, add your name up top to the list of authors in 16.1.  You're already in AUTHORS right?